tests/8_breeding.js

```diff
@@ -0,0 +1,187 @@
+  let tokenData = res.events.DataUpdated.returnValues.newData;
+  console.log({ tokenId, tokenData });
+
+  console.log('   Approving..');
+  await deployedToken.methods.approve(contracts.exitHandler.options.address, tokenId).send({ from: minter });
+  console.log('   Depositing..');
+  await contracts.exitHandler.methods.depositBySender(tokenId, nstColor)
+    .send(
+      {
+        from: minter,
+        gas: 2000000,
+      }
+    );
+
+  console.log('    advanceBlocks');
+  await node.advanceUntilChange(web3);
+
+  let unspents = (await node.send('plasma_unspent', [minter, nstColor])).result;
+  assert.equal(unspents[0].output.data, tokenData, 'tokenData should match');
+
+  const script = Buffer.from(BreedingCondition, 'hex');
+  const scriptHash = ethUtil.ripemd160(script);
+  const spAddr = `0x${scriptHash.toString('hex')}`;
+  // XXX: add support for approval in ERC1949
+  let transferTx = Tx.transfer(
+    [
+      new Input({
+        prevout: new Outpoint(unspents[0].outpoint.slice(0, -2), 0),
+      }),
+    ],
+    [
+      new Output(
+        unspents[0].output.value,
+        spAddr,
+        unspents[0].output.color,
+        unspents[0].output.data,
+      ),
+    ],
+  );
+  transferTx.signAll(minterPriv);
+  await node.sendTx(transferTx);
+
+  const unspentsLEAP = (await node.send('plasma_unspent', [minter, 0])).result;
+  const unspentsSp = (await node.send('plasma_unspent', [spAddr, nstColor])).result;
+  const depositInput = new Outpoint(unspentsSp[0].outpoint.slice(0, -2), 0);
+
+  let gasInput;
+  unspentsLEAP.forEach(
+    (unspent) => {
+      if (unspent.output.value > 1000000000) {
+        gasInput = new Outpoint(unspent.outpoint.slice(0, -2), parseInt(unspent.outpoint.slice(-2), 16));
+      }
+    }
+  );
+  const condTx = Tx.spendCond(
+    [
+      new Input({
+        prevout: gasInput,
+        script,
+      }),
+      new Input({
+        prevout: depositInput,
+      }),
+    ],
+    []
+  );
+  const msgData =
+    '0x689dcafc' + // function called
+    `${tokenId.replace('0x', '')}000000000000000000000000${
+    minter.replace('0x','').toLowerCase()
+    }0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000` +
+    `${deployedToken.options.address.replace('0x', '').toLowerCase()}`;
+
+  condTx.inputs[0].setMsgData(msgData);
+  condTx.signAll(minterPriv);
+
+  let computedOutputs = (await node.send('checkSpendingCondition', [condTx.hex()])).result;
+  computedOutputs.outputs.forEach(
+    (output) => {
+      condTx.outputs.push(Output.fromJSON(output));
+    }
+  );
+  condTx.signAll(minterPriv);
+
+  console.log('sending breeding condition');
+  await node.sendTx(condTx);
+
+  unspents = (await node.send('plasma_unspent', [minter, nstColor])).result;
+  console.log('-----------------unspents--------------');
+  console.log(unspents);
+  console.log('what a bullshit, circumventing proof bug with youngestInputIndex > 0');
```

Comment:
is there an issue about this somewhere?

Reply:
Yep: https://github.com/leapdao/leap-contracts/issues/184